{
	"header": {
		"line": "CVE-2018-14615 (An issue was discovered in the Linux kernel through 4.17.10. There is  ...)",
		"name": "CVE-2018-14615",
		"description": "(An issue was discovered in the Linux kernel through 4.17.10. There is  ...)"
	},
	"annotations": [
		{
			"line": "\t- linux 4.19.9-1",
			"type": "package",
			"package": "linux",
			"kind": "fixed",
			"version": "4.19.9-1"
		},
		{
			"line": "\t[stretch] - linux <not-affected> (Vulnerable code not present)",
			"type": "package",
			"release": "stretch",
			"package": "linux",
			"kind": "not-affected",
			"description": "Vulnerable code not present"
		},
		{
			"line": "\t[jessie] - linux <not-affected> (Vulnerable code not present)",
			"type": "package",
			"release": "jessie",
			"package": "linux",
			"kind": "not-affected",
			"description": "Vulnerable code not present"
		},
		{
			"line": "\tNOTE: https://bugzilla.kernel.org/show_bug.cgi?id=200421",
			"type": "NOTE",
			"description": "https://bugzilla.kernel.org/show_bug.cgi?id=200421"
		}
	]
}
